packages/filters/filter-color-matrix/src/ColorMatrixFilter.ts

```diff
@@ -141,6 +141,30 @@ export class ColorMatrixFilter extends Filter
         this._loadMatrix(matrix, multiply);
     }
 
+    /**
+     * Sets each channel on the diagonal of the color matrix.
+     * This achieves a tinting effect.
```

Comment:
Would be nice to explain why you'd use this instead of `tint` property found on most display objects? What's the advantage.

Reply:
Container doesn't have tint. Only sprites do. That's the whole idea behind this but I could make it more clear.

Reply:
Sprite, Text, Graphics, Mesh have some tint implementation. Some additional context here for users would be this tint could support any Container object.

Reply:
Added a small text to explain why and when you should use tint.